packages/configurationmap/style.ts

```diff
@@ -0,0 +1,3 @@
+import { css } from "emotion";
```

Comment:
Do we need this file?

Reply:
I'd be in favor of removing this until we actually have styles